app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_save :post_counter_update
+
+  def post_counter_update
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ ] You did well using one of the Rails Active callbacks to call the `post_counter_update`. it would also be nice if you could make the `post_counter_update` method private.